test/irb/test_init.rb

```diff
@@ -50,6 +50,27 @@ def test_rc_file
       end
     end
 
+    def test_rc_file_loading_order
+      tmpdir = @tmpdir
+      Dir.chdir(tmpdir) do
+        # tmpdir is ENV["HOME"], so another dir is needed
+        subdir = "#{tmpdir}/mydir"
+        FileUtils.mkdir_p("#{tmpdir}/mydir")
```

Comment:
Then we can just use the above local here:

```
FileUtils.mkdir_p(project_dir)
```

Reply:
Done, thank you!